apps/staking/src/components/OracleIntegrityStaking/index.tsx

```diff
@@ -517,14 +518,7 @@ const Publisher = ({
           </Meter>
         </PublisherTableCell>
         <PublisherTableCell className="text-center">
-          <div>
-            {calculateApy(
```

Comment:
I think the APY we display here should be the forecasted APY for the next epoch, if there are no more changes

Reply:
done